src/EventSubscriber/SimplesamlSubscriber.php

```diff
@@ -57,17 +57,17 @@ public function redirectToSimplesamlLogin(GetResponseEvent $event) {
 
     $request = $event->getRequest();
     $config = \Drupal::config('os2web_simplesaml.settings');
-
     // Only redirect if we are on redirect triggering page.
-    if ($request->getRequestUri() == $config->get('redirect_trigger_path')) {
+    $patterns = str_replace(',', "\n", $config->get('redirect_trigger_path'));
+    if (empty($patterns) || \Drupal::service('path.matcher')->matchPath($request->getRequestUri(), $patterns)) {
```

Comment:
it seems it must be && instead of ||
```suggestion
    if (empty($patterns) && \Drupal::service('path.matcher')->matchPath($request->getRequestUri(), $patterns)) {
```